### DISCUSSION TITLE
Deep nested components and sharedValue

### DISCUSSION THREAD

**Discussion by RomanSuvorov** [software-mansion/react-native-reanimated#2470] [Q&A]
Hello everybody,
Can i save SharedValue to store (e.x. Redux store) to use this in different deep nested components;

**Comment by RomanSuvorov** [comment]
Any suggestions?

**Reply by Noitidart** [reply]
Yea it would work fine. I haven't used redux store though, I like my redux store serializable. I use context for this purpose.
